index.js

```diff
@@ -1,14 +1,23 @@
-import vc from '@digitalbazaar/vc';
-import contexts from '@digitalbazaar/vc/lib/contexts';
+import {defaultDocumentLoader, issue} from '@digitalbazaar/vc';
 import {extendContextLoader} from 'jsonld-signatures';
 import ed25519Context from 'ed25519-signature-2020-context';
+import * as examples1Context from '@digitalbazaar/credentials-examples-context';
```

Comment:
this would be a linter error in the DB system as all comes from multiple or single imports.

Reply:
Good to know. I can add our eslint rules later and do some cleanup. You OK to proceed, though?